Public/Src/Tools/ContentPlacementAnalysisTools/ContentPlacementAnalysisTools.Extraction/CPResources/constants.Designer.cs

```diff
@@ -0,0 +1,72 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000

```

Comment:
Please do not check this file in. Standard MsBuild puts generated files in the source tree, but in Bxl we store these properly in the object tree. 

Our filosophy is that no generated files should ever be checked in :)